pkg/sql/schemachanger/sctest/testdata_path_helpers.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package sctest
+
+import (
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/cockroachdb/cockroach/pkg/build/bazel"
+	"github.com/cockroachdb/cockroach/pkg/testutils"
+)
+
+// testdataPath provides a path either relative to the current working
+// directory or absolute to the input which is a relativePath compared
+// to the root of the workspace.
+func testdataPath(t *testing.T, relativePath string) string {
+	if bazel.BuiltWithBazel() {
+		return testutils.RewritableDataPath(t, relativePath)
+
+	}
+	callers := make([]uintptr, 512)
```

Comment:
nit: magic number, define constant instead

Reply:
This code is now gone. `testutils.RewritableDataPath` was what I wanted all along.